plugin/builtin/replicacounttransformer/ReplicaCountTransformer_test.go

```diff
@@ -218,7 +218,7 @@ spec:
 		t.Fatalf("No match should return an error")
 	}
 	if err.Error() !=
-		"resource with name service does not match a config with the following GVK [~G_~V_Deployment]" {
+		"resource with name service does not match a config with the following GVK [Deployment.[noVer].[noGrp]]" {
```

Comment:
Can we get rid of the surrounding brackets now that we're using them internally?

Reply:
The surrounding brackets are not something we've added, but get added because we are outputting a list of gvks:

https://github.com/kubernetes-sigs/kustomize/blob/3608f335fd419a567d8e62e0de5118421e424331/plugin/builtin/replicacounttransformer/ReplicaCountTransformer.go#L61-L62